Automattic/jetpack
# Protect: Add WAF subheadings

#### Description
Adds various subheadings to the primary WAF header component depending on the combination of settings enabled.

## Proposed changes:

* Adds subcomponents and logic for the display of the subheading

### Other information:


## Jetpack product discussion


- 1201069996155224-as-1203846677289621
- p6TEKc-6Lx-p2

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Start up your Jurassic Tube and enable Protect
* Jump to the Firewall page
* Toggle the various settings available and ensure that each subheading displayed is appropriate for the combination enabled (as per the designs), for each of the following:
   * With no plan
   * After upgrading
   * After downgrading
   * After manually deleting the rules directory
   * After manually disabling the WAF module via Jetpack Settings (not yet accounted for)
